### ISSUE TITLE
How to continue rebasing after an edit action?

### ISSUE THREAD

**Issue by KentVu** [ForkIssues/Tracker#895]
I use interactive rebase to edit a commit (using edit action). After amending that commit, I found no option to continue rebasing (I was searching for something like "continue rebase" but not found). Is this actually implemented?

Related: #201 #179

**Comment by DanPristupov** [comment]
There is 'Continue rebase' button, please check the last screenshot.

I'll show RI with amend/edit step by step:

Start RI:
<img width="500" alt="Screenshot 2020-02-04 at 11 12 38" src="https://user-images.githubusercontent.com/618115/73735244-42f3b100-473f-11ea-964a-9c035f7535aa.png">

Select Edit:
<img width="500" alt="Screenshot 2020-02-04 at 11 13 10" src="https://user-images.githubusercontent.com/618115/73735290-556dea80-473f-11ea-9c95-2501b7d0d57e.png">

Amend the commit:
<img width="500" alt="Screenshot 2020-02-04 at 11 13 36" src="https://user-images.githubusercontent.com/618115/73735341-661e6080-473f-11ea-9adc-0b5a82f168e7.png">

After amending, continue rebase:
<img width="500" alt="Screenshot 2020-02-04 at 11 13 50" src="https://user-images.githubusercontent.com/618115/73735363-6dde0500-473f-11ea-8c47-fcccb9b088d7.png">

**Comment by KentVu** [comment]
Mine just show the commit button after amending.
Fork version is 1.0.89. 
I'm editing the first commit in the series.

<img width="856" alt="Screen Shot 2020-02-05 at 10 21 16" src="https://user-images.githubusercontent.com/3026524/73802635-96c6ce80-4801-11ea-94ec-9d337e4596cc.png">

**Comment by DanPristupov** [comment]
> Mine just show the commit button after amending.

Probably because you have unstaged files. Try to press commit, it should continue rebase.

**Comment by KentVu** [comment]
Pressing commit button caused error:
```
$ git commit --file=/var/folders/yk/dp1g14v15fgfy603dr3b_xhr0000gp/T/5B3E13DA-CE78-45CD-A4E5-F490B20633B6

interactive rebase in progress; onto 9ad4cac40
Last command done (1 command done):
   e 2a714e77d Bug #19。。。文言。
Next command to do (1 remaining command):
   p 042500f68 Bug #1......p start).
You are currently splitting a commit while rebasing branch 'A...162' on '9ad4cac40'.

Changes not staged for commit:
	modified:   ... (modified content, untracked content)

Untracked files:
	KD.../.vscode/
	....

no changes added to commit
```

<img width="677" alt="Screen Shot 2020-02-05 at 17 58 48" src="https://user-images.githubusercontent.com/52944767/73826969-0e1c5280-4842-11ea-8276-85031454f2a7.png">

The `commit` button is not changing.

PS: When I try `git rebase --continue` in terminal it gives me the error:
```
$ git rebase --continue 
error: could not read '.git/rebase-merge/quiet': No such file or directory
```
Which I have to `touch .git/rebase-merge/quiet` to make it success.

My git version:
```
$ git --version
git version 2.20.1 (Apple Git-117)
$ which git
/usr/bin/git
```

**Comment by DanPristupov** [comment]
I checked the source code. Fork is confused by the fact that you have modified files (in your case it's a submodule). We should probably ignore submodules during this test.

Can you try to stash your changes in the submodule and then rebase please?

**Comment by KentVu** [comment]
Hi Dan,
I have confirmed your case: After stashing changes AND deleting all new (untracked) files in the submodule the `continue rebase` button appeared.
<img width="851" alt="Screen Shot 2020-02-06 at 13 36 03" src="https://user-images.githubusercontent.com/3026524/74133444-eeaf6c00-4c2b-11ea-8e40-e4fc74f7b02b.png">

Hope this helps!

**Comment by DanPristupov** [comment]
We should probably ignore dirty submodules and allow to continue the rebase. I will think more about this and most likely fix this in Fork 1.0.90.

**Comment by KentVu** [comment]
I think it's better displaying the `continue rebase` button all the time.
The `commit` button should only appear when editing the commit message or when `Amend`ing.
Or just simply displaying both the two buttons isn't bad idea don't you think?
